tools/integration_tests/util/setup/yaml-config.go

```diff
@@ -0,0 +1,29 @@
+package setup
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	config2 "github.com/googlecloudplatform/gcsfuse/internal/config"
+	"gopkg.in/yaml.v2"
+)
+
+func YAMLConfig(createEmptyFile bool) (filepath string) {
+	config := config2.MountConfig{
```

Comment:
Writing a method for each and every property will become difficult. 
Instead of that, can we have a method which takes WriteConfig ?